src/js/CheckModule.js

```diff
@@ -0,0 +1,61 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var CheckModule = React.createClass({
+  render: function() {
+    return (
+      <div>
+        <RetainedButton onCheckedStatusChanged={this.props.onCheckedStatusChanged} /> // rendering the buttons and passing the call backs as props
+        <ReplacedButton onCheckedStatusChanged={this.props.onCheckedStatusChanged} />
+        <WrongButton onCheckedStatusChanged={this.props.onCheckedStatusChanged} />
+        <UncheckButton onCheckedStatusChanged={this.props.onCheckedStatusChanged} />
+      </div>
+    );
+  }
+});
+
+var RetainedButton = React.createClass({
+  handleClick: function() {
+    this.props.onCheckedStatusChanged("RETAINED");
+  },
+  render: function() {
+    return (
+      <button onClick={this.handleClick}>Retained</button>
+    );
+  }
+});
```

Comment:
Put these buttons in separate files because they are modules of their own.

Import them here afterwards.